client/src/components/CheckList.js

```diff
@@ -35,9 +34,10 @@ function CheckList({lawsuits, myMoment, setMyMoment}){
     const displayDemurrerChecklist = lawsuits.map((lawsuit)=>{
         return lawsuit.deadlines.map((deadline)=>{
             const deadlineDate = moment(deadline.deadline)
-            console.log(deadlineDate)
-            // COMPARE line 17 myMoment to line 38 deadlineDate to resolve logic on line 40.  This controls conditional rendering
-            if (myMoment < deadlineDate.subtract(15, "days")) {
+            console.log(myMoment, "MY MOMENT")
```

Comment:
I presume the console logs are to help understand what's happening in the feature, can leave these in for now but long term should eventually get ride of them